src/vsr/replica.zig

```diff
@@ -5262,7 +5262,7 @@ pub fn ReplicaType(
 
         fn transition_to_recovering_head(self: *Self) void {
             assert(self.status == .recovering);
-            assert(self.view >= self.log_view);
+            assert(self.view == self.log_view);
```

Comment:
double checking if intended

Reply:
Yeah, it's intentional. The reason is that if `view > log_view`, then we would have a durable DVC in our superblock, in which case our op is always "certain", since we will just truncate everything after it anyway.
When `view == log_view` we have a durable SV in our superblock. Since we might have prepared ops after that, we can't be certain about the op.